Modified the Angular regex

jaril opened the pull request:
Fixes #7901

### Summary of Changes

Modified `getLibraryFromUrl()` so that it only matches Angular's core framework sub folder, not its components.
- Matches `/turbo_modules/@angular/core@7.0.1/bundles/core.umd.js`
- Doesn't match `/~/src/app/hello.component.ts`


Note that in the image below, `ngOnInit` (which is defined in hello.component.ts) isn't collapsed into the Angular frame group:

<img width="387" alt="screen shot 2019-02-13 at 9 35 38 am" src="https://user-images.githubusercontent.com/15959269/52719301-61333080-2f73-11e9-8214-d41da12eb9c7.png">

jaril commented:
Made this more comprehensive to match all of Angular's framework code. This now matches the following cases:

- AngularJS (AngularJS 1.x) which usually bundled angular code as `angular.js` or `angular.min.js`. Examples are [Google Family Link](https://families.google.com/familylink/) and [United Jetstream](https://jetstream.united.com/#/public-landing).
- Angular (AngularJS 2.x) which bundles Angular code within [directories](https://angular.io/api) (e.g. core, common). For example, most of Angular's core functionality is implemented by `/@angular/core@7.2.4/bundles/core.umd.js`.

darkwing commented on src/utils/pause/frames/getLibraryFromUrl.js:
```diff
@@ -78,7 +78,13 @@ const libraryMap = [
+    pattern: new RegExp(
+      "angular\\.|" +
+        "angular.*" +
+        "(animations|common|core|elements|forms|platform-browser|" +
```
I'm slightly worried that using these specific strings will cause a maintenance issue, in that every time they rename a file or add one, we'll need to make updates.  Can we do a `angular.([A-z+])` type of regex?  It's more generic but I doubt many people purposely prefix the specific JS with `angular.`

jaril replied:
Good call - switched this out so that it's much simpler :)

jaril commented:
Changed this so that all it does is it doesn't recognize Angular Components as framework code, which was the issue brought up in the first place. 

Added unit tests to that effect, and to check framework URLs for both AngularJS and Angular (1.x and 2.x)
